### ISSUE TITLE
Duplicate owner entries are not filtered out in the blob view

### ISSUE THREAD

**Issue by leonore** [sourcegraph/sourcegraph#49420]
![Image](https://user-images.githubusercontent.com/29406849/225355067-20307ad4-fe32-4292-bf65-2456c6d5ee1a.png)

the entry in question:
```
* @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e @a @b @c @d @e
```

**Comment by cbart** [comment]
There is a wider problem here of unification generally in the resolver codebase.

**Comment by cbart** [comment]
Note: For resolution of git commit signals, email is sometimes a github-encoded-username like 68532117+abeatrix@users.noreply.github.com

**Comment by sashaostrikov** [comment]
fixed in 5.0
